Exiled.API/Features/Player.cs

```diff
@@ -953,6 +955,16 @@ public float Stamina
             set => StaminaStat.CurValue = value;
         }
 
+        /// <summary>
+        /// Gets or sets the stamina usage multiplier.
+        /// </summary>
+        public float StaminaUsageMultiplier { get; set; } = 1f;
+
+        /// <summary>
+        /// Gets or sets the stamina regen multiplier.
+        /// </summary>
+        public float StaminaRegenMultiplier { get; set; } = 1f;
+
```

Comment:
Before someone else merge this things

 should this be inside Player::Role.As<FpcRole>() ?

Reply:
Mb no, To example, IsUsingStamina not in Role class, so StaminaUsageMultiplier and StaminaRegenMultiplier should be at Player class too

Reply:
that why i ask actually the good part of making it inside Role it's than it's would be reset everytime player change Role

Reply:
I made some changes